ci/tests.py

```diff
@@ -30,13 +31,18 @@
 
 
 def _availability_zones():
-    api_manager = manager.APIManager()
     try:
+        api_manager = manager.APIManager()
         zones = api_manager.availability_zone_client.list_availability_zones()
         info = zones['availabilityZoneInfo']
         return {zone['zoneName'] for zone in info}
+    except:
+        return {}
     finally:
-        api_manager.cleanup_credentials()
+        try:
```

Comment:
Instead of adding a new `try - catch` block you could make `api_manager = None` at the start of the function, and just add an `if` statement in the `finally` block.